### ISSUE TITLE
Add site tracking capabilities, i.e., google analytics or some such service

### ISSUE THREAD

**Issue by wsbctechnicalblog** [wsbctechnicalblog/wsbctechnicalblog.github.io#220]
We are still running blind with WSBC technical blog, not knowing the telemetry. It is urgent to enable the site to collect telemetry so that we could apply validated learning and gain much needed insights into the level of traction each blog post garners.

@wpschaub, @erwsch1 can we add this project to our upcoming hackathon?

**Comment by wpschaub** [comment]
Good idea. Please add it to the list of hackathon ideas document on our portal.

**Comment by wsbctechnicalblog** [comment]
Done.
